source/core/NumericString.cpp

```diff
@@ -1291,6 +1291,44 @@ IntMax ScanIntValues(char formatChar, char* beginPointer, char** endPointer)
     return intValue;
 }
 
+static const char *validTrueValues[] = { "y", "t", "true", "on", "yes" };
+static const char *validFalseValues[] = { "n", "f", "false", "off", "no" };
+
+Boolean TryScanBooleanValue(char* beginPointer, char** endPointer, Boolean* value)
```

Comment:
The return of a Boolean success flag seems an insufficient reason to me for putting the word "Try" in the function name.  We don't have this kind of convention anywhere else.